cli/cobra.go

```diff
@@ -230,7 +245,7 @@ Options:
 Management Commands:
 
 {{- range managementSubCommands . }}
-  {{rpad .Name .NamePadding }} {{ if isFirstLevelCommand .}}{{commandVendor .}} {{ end}}{{.Short}}
+  {{rpad (decoratedCommandName .) (add .NamePadding 1)}}{{.Short}}{{ if isPluginFirstLevelCommand .}} {{commandVendorAndVersion .}}{{ end}}
```

Comment:
With my suggestions above, this could probably be 

```suggestion
  {{rpad (decoratedCommandName .) (add .NamePadding 1)}}{{.Short}}{{ if isPlugin .}} {{vendorAndVersion .}}{{ end}}
```

Or (if we make `vendorAndVersion` return nothing in the non-plugin case);

```suggestion
  {{rpad (decoratedCommandName .) (add .NamePadding 1)}}{{.Short}} {{vendorAndVersion .}}
```

Reply:
Done